sdk/src/client/node_api/core/routes.rs

```diff
@@ -103,6 +103,19 @@ impl ClientInner {
             .await
     }
 
+    /// Returns the totally available Mana rewards of an account or delegation output decayed up to epochEnd index
+    /// provided in the response.
```

Comment:
Maybe also include the note in the description? It can be helpful for education? 🤷🏻‍♂️

Reply:
In which description? :sweat_smile:

Reply:
> description: >-
        Returns the totally available Mana rewards of an account or delegation output up until the current epoch index. 
        Note that rewards for an epoch only become available at the beginning of the next epoch. 
        If the end epoch of a staking feature is equal or greater than the current epoch, the rewards response will not include the potential future rewards for those epochs.
        epochStart and epochEnd indicates the actual range for which reward value is returned and decayed for.

Reply:
Ah like this, sure